app/controllers/concerns/search.rb

```diff
@@ -7,8 +7,16 @@ module Search
   included do
     def scope_search
       result = policy_scope(resource)
+      escape_references_search
       result = result.search_for(params[:search]) if params[:search].present?
       result.paginate(page: pagination_offset, per_page: pagination_limit)
     end
+
+    def escape_references_search
+      return unless params[:search]&.match(/reference=/)
```

Comment:
This regex isn't quite right. It needs to be able to match any [valid query](https://github.com/wvanbergen/scoped_search/wiki/Query-language) for a string field.

```suggestion
      return unless params[:search]&.match(/reference\s?(=|!=|<>|~|!~|\^|!\^)\s?/)
```

Reply:
This is also problematic if users don't specify `reference=`. Scoped search is meant to search every searchable field when just the keyword is given.

Reply:
We can make it `only_explicit` though, and allow only `=` - otherwise we'll have to take into account every possibility in the API for what I believe is likely a corner case

Reply:
There's no reason an API user shouldn't be able to search by references as a generic keyword; disabling that functionality with `only_explicit` just for this corner case doesn't seem like the best way to solve this. Having special behavior for this field will make the API more difficult to document and maintain. API users, including our own frontend, should just properly double-quote such search terms.

Reply:
By the way, this search term is a collection, so users must use the 'includes' operator `^` and cannot use the string comparison operators mentioned above.